lib/executor/vu_handle.go

```diff
@@ -39,6 +39,9 @@ type vuHandle struct {
 	parentCtx context.Context
 	getVU     func() (lib.InitializedVU, error)
 	returnVU  func(lib.InitializedVU)
+	exec      string
+	env       map[string]string
+	tags      map[string]string
```

Comment:
Same here - a pointer to the `Config` (or `BaseConfig`) will probably be better than all of these in every VU handle

Reply:
Should be resolved by f7e0214c.